utils/ErrorOutput.go

```diff
@@ -14,7 +14,7 @@ type ValidationError struct {
 
 func ClientError(statusCode int, body string) (events.APIGatewayProxyResponse, error) {
 	//check if the status sent is in the range of 400 and 500.
-	if !(statusCode >= http.StatusBadRequest && statusCode < http.StatusInternalServerError) {
+if !(statusCode >= http.StatusBadRequest && statusCode < http.StatusInternalServerError) {
```

Comment:
I think its better to keep the space indent

Reply:
Done